Pod/Classes/Forms/UIView+Lookup.swift

```diff
@@ -16,41 +16,27 @@ extension UIView {
     /// - parameter includeNonInteractable: Return views that do not have user interaction enabled. The default is false.
     ///
     /// - returns: An array of views, of type T, sorted by y position.
-    public func lookupSortedViews<T: UIView>(includeHidden: Bool = false, includeNonInteractable: Bool = false) -> [T] {
-        if let textField = self as? T {
+    final public func sortedSubviews<T: UIView>(ofType type: T.Type, includeHidden: Bool = false, includeNonInteractable: Bool = false) -> [T] {
+        if let view = self as? T {
             if (hidden == false || (includeHidden && hidden == true)) &&
                 (userInteractionEnabled == true || includeNonInteractable && userInteractionEnabled == false) {
-                return [textField]
+                return [view]
             }
         }
         var views: [T] = Array()
         for subview in subviews {
-            let results: [T] = subview.lookupSortedViews(includeHidden, includeNonInteractable: includeNonInteractable)
+            let results = subview.sortedSubviews(ofType: T.self, includeHidden: includeHidden, includeNonInteractable: includeNonInteractable)
             views.appendContentsOf(results)
         }
-        views.sortInPlace { (textField, otherTextField) -> Bool in
-            let center = convertPoint(textField.center, fromView: textField.superview)
-            let otherCenter = convertPoint(otherTextField.center, fromView: otherTextField.superview)
+        views.sortInPlace { (view, otherView) -> Bool in
+            let center = convertPoint(view.center, fromView: view.superview)
+            let otherCenter = convertPoint(otherView.center, fromView: otherView.superview)
             return center.y < otherCenter.y
         }
         return views
     }
-    
-    public func lookupNextView<T: UIView>(currentView: T, includeHidden: Bool = false, includeNonInteractable: Bool = false) -> T? {
-        let views: [T] = self.lookupSortedViews(includeHidden, includeNonInteractable: includeNonInteractable)
-        guard let index = views.indexOf(currentView) else { return nil }
-        let nextIndex = index.successor()
-        return (views.indices.contains(nextIndex)) ? views[nextIndex] : nil
-    }
-    
-    public func lookupPreviousView<T: UIView>(currentView: T, includeHidden: Bool = false, includeNonInteractable: Bool = false) -> T? {
-        let views: [T] = self.lookupSortedViews(includeHidden, includeNonInteractable: includeNonInteractable)
-        guard let index = views.indexOf(currentView) else { return nil }
-        let previousIndex = index.predecessor()
-        return (views.indices.contains(previousIndex)) ? views[previousIndex] : nil
-    }
 
-    public func lookupParentView<T: UIView>() -> T? {
+    final public func nearestSuperview<T: UIView>(ofType type: T.Type) -> T? {
```

Comment:
#pp this naming is confusing, I like `lookupParentView` better or just `parentView` or `container`